lib/MT.pm

```diff
@@ -33,7 +33,7 @@ our $plugins_installed;
 BEGIN {
     $plugins_installed = 0;
 
-    ( $VERSION, $SCHEMA_VERSION ) = ( '6.7', '6.0022' );
+    ( $VERSION, $SCHEMA_VERSION ) = ( '6.7', '6.0023' );
```

Comment:
@charsbar 
Do we need to raise the schema version of the core in this PR?

Reply:
@usualoma I agree it's weird but I saw a strange error without it (because of core_remove_news_widget_cache sending children hup signal?).

Reply:
Oh, I get it!